examples/gke/gke_cluster_gitops/main.tf

```
@@ -0,0 +1,32 @@
+resource "castai_gke_cluster" "my_castai_cluster" {
```

```suggestion
resource "castai_gke_cluster" "this" {
```

We will have only one resource with that name :)

---

Valid point, thanks! 🚀